posthog/api/sessions_filter.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict
+
+from rest_framework import serializers, viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from posthog.api.routing import StructuredViewSetMixin
+from posthog.api.user import UserSerializer
+from posthog.mixins import AnalyticsDestroyModelMixin
+from posthog.models import SessionsFilter, Team
+from posthog.permissions import ProjectMembershipNecessaryPermissions
+
+
+class SessionsFilterSerializer(serializers.ModelSerializer):
+    created_by = UserSerializer(required=False, read_only=True)
+
+    class Meta:
+        model = SessionsFilter
+        fields = ["id", "name", "created_by", "created_at", "updated_at", "filters"]
+        read_only_fields = [
+            "id",
+            "created_by",
+            "created_at",
+            "updated_at",
+        ]
+
+    def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> SessionsFilter:
+        request = self.context["request"]
+        return SessionsFilter.objects.create(team=request.user.team, created_by=request.user, **validated_data,)
```

Comment:
Think it's worth adding instrumentation on the create and update methods (same as we do with actions, feature flags, ...)

Reply:
Note, no updates currently happen. For updates would need to add dead code for this.
Added instrumentation for create.